.github/workflows/cont_integration.yml

```
@@ -50,7 +50,12 @@ jobs:
       run: rustup update
     - name: pin dependencies
       if: matrix.rust.version == '1.57.0'
-      run: cargo update -p log --precise 0.4.18 && cargo update -p rustls:0.21.2 --precise 0.21.1 && cargo update -p time:0.3.15 --precise 0.3.13
+      run: |
+        cargo update -p log --precise 0.4.18 && 
+        cargo update -p rustls:0.21.5 --precise 0.21.1 && 
+        cargo update -p time:0.3.15 --precise 0.3.13 &&
+        cargo update -p hyper-rustls:0.24.1 --precise 0.24.0 &&
+        cargo update -p tempfile:3.7.0 --precise 3.6.0
```

The `&&` aren't needed when put on new lines like this.

```suggestion
        cargo update -p log --precise 0.4.18
        cargo update -p rustls:0.21.5 --precise 0.21.1
        cargo update -p time:0.3.15 --precise 0.3.13
        cargo update -p hyper-rustls:0.24.1 --precise 0.24.0
        cargo update -p tempfile:3.7.0 --precise 3.6.0
```